pkg/subnet/public.go

```diff
@@ -626,6 +626,8 @@ func (d *PublicDeployer) loadWallet(subnetIDs ...ids.ID) (*primary.Wallet, error
 
 func (d *PublicDeployer) CleanCacheWallet() {
 	d.wallet = nil
```

Comment:
question: i'm not seeing this cached `d.wallet` is being used in this file. it is all a local `wallet` var is being used in other functions. for my understanding: what does `d.wallet` do in this file?

Reply:
It is returned by loadCacheWallet, which is called by many functions here.

Reply:
ok got it. missed the part that `d.wallet` is returned.

Reply:
this code needs cleanup, and using d.wallet instead of loadCacheWallet is one of those. probably that can wait until refactoring this into SDK, but feel free to make a PR for this.